### ISSUE TITLE
Error while running bin/quick-start-offline.sh

### ISSUE THREAD

**Issue by tekbird** [apache/pinot#2346]
I followed the instructions at https://github.com/linkedin/pinot to test the pinot. Although build was success, I got the following error while starting offline quick start:

[root@localhost pinot-0.016-pkg]# bin/quick-start-offline.sh
***** Starting Zookeeper, controller, broker and server *****
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/commons/logging/Log
        at com.linkedin.pinot.tools.admin.command.StartControllerCommand.execute(StartControllerCommand.java:138)
        at com.linkedin.pinot.tools.admin.command.QuickstartRunner.startControllers(QuickstartRunner.java:87)
        at com.linkedin.pinot.tools.admin.command.QuickstartRunner.startAll(QuickstartRunner.java:124)
        at com.linkedin.pinot.tools.Quickstart.execute(Quickstart.java:164)
        at com.linkedin.pinot.tools.Quickstart.main(Quickstart.java:227)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.logging.Log
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:338)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 5 more

Can someone tell me what is wrong here?

**Comment by jagarwal0424** [comment]
Hi tekbird,

I am trying to build Pinot using below steps, but i don't find  pinot-distribution/target/pinot-0.016-pkg folder after doing successful mvn. How did manage that? 

git clone https://github.com/linkedin/pinot.git
cd pinot
mvn install package  -DskipTests
cd pinot-distribution/target/pinot-0.016-pkg
chmod +x bin/*.sh

**Comment by tekbird** [comment]
If you do mvn install step, it should create the pkg file

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by jagarwal0424** [comment]
Hi tekbird,

Thanks for your response, 
mvn install step seem successfull but still pinot-distribution folder does not contain required pkg. Please see attached log when you have time.

[mvn_error.txt](https://github.com/linkedin/pinot/files/1669871/mvn_error.txt)

**Comment by WicketWarrick** [comment]
Hi tekbird,

I have the exact same problem and error. I tried adding the following dependency to the pom.xml file:
`<dependency>
        <groupId>commons-logging</groupId>
        <artifactId>commons-logging</artifacts>
        <version>1.2</version>
      </dependency>
` 
But I still get the same result. I did some research on stackoverflow and others, but most of the time they just say that it should work if the dependency is added to the pom.xml file, if the commons-logging-1.2.jar is in the lib folder, and if that .jar contains the classes that are causing the error (it's almost always on the LogFactory class on stackoverflow threads). 

Are you still stuck on the same error ? If not, I would love to know how you did to fix it.
